style.css/general.css

```diff
@@ -0,0 +1,33 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  line-height: 29px;
+}
+
+.content {
+  flex: 1 0 auto;
+}
+
+.site-wrapper {
+  max-width: 1200px;
+  margin: 0 16px;
```

Comment:
Why did you do this? After this change is all fine with the design? I asked to add padding: 0 16px;
After removing margin auto all content is on the left side.

Reply:
That's my bad I fixed it.